package.json

```diff
@@ -259,6 +259,17 @@
           },
           "markdownDescription": "Disable Qute validation for the given file name patterns.\n\nExample:\n```\n[\n \"**/*items.qute.*\"\n]```.",
           "scope": "resource"
+        },
+        "qute.validation.undefinedObject": {
```

Comment:
After debugging the Qute LS which crashes, it's because the preference name is not correct.

It should be named `qute.validation.undefinedObject.severity`